src/components/Nav/index.js

```diff
@@ -295,32 +392,9 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
       to: "/developers/local-environment/",
       shouldDisplay: contentVersion > 1.1,
     },
-    {
-      text: "page-glossary",
-      to: "/glossary/",
-      shouldDisplay: contentVersion > 1.1,
-    },
   ]
   let mobileLinkSections = cloneDeep(linkSections)
 
-  // If contentVersion includes EDN (>1.1), strip out Developers links
```

Comment:
Do we want to remove this logic? I don't think so but @ryancreatescopy please confirm. My understanding is we want to keep the single "Developers" link in main nav, since there's a subnav. Currently Developers now has both the dropdown & the subnav, which feels awkward:
![Image 2020-11-11 at 3 00 45 PM](https://user-images.githubusercontent.com/8097623/98874245-bb09a880-242e-11eb-9596-460629c35e64.png)

Reply:
Yeah either way... not my call of course, but I'll just mention I've found it nice having that dropdown there while testing. 
I like being able to toggle that from any page and quickly see the sub-sections (just like with Use Ethereum and Learn)... otherwise it requires navigation to load the Developers page before you can see the Docs/Tutorials/etc options.
Example: I want to get to the tutorials from some random page, but I forget if it's part of Learn, or Developers. From any page (without leaving that page) I can peak at both dropdown menus to confirm before navigating to Developers.

Totally get your point in the screenshot though, makes it look a little weird if you're already on Developers and click that 🤷‍♂️

Reply:
@samajammin I added it back in. I don't think it's that harmful for users to have both menus while on `/developers/` and figured it's a better experience that allows users to dive in to a page like tutorials, for example, from anywhere on the site. Happy to discuss this further if you think it creates a bad experience.